app/repository/document.py

```diff
@@ -125,7 +125,7 @@ def _document_tuple_from_dto(db: Session, dto: DocumentCreateDTO) -> CreateObjec
     phys_doc = PhysicalDocument(
         id=None,
         title=dto.title,
-        source_url=cast(str, dto.source_url) if dto.source_url is not None else None,
+        source_url=str(dto.source_url) if dto.source_url is not None else None,
```

Comment:
Can we mark a TODO on this ... and that is to do some more verification here - unless you want to do that now ... it would involve ensuring that if its not None - then its a valid url

Reply:
[PDCT-865](https://linear.app/climate-policy-radar/issue/PDCT-865/add-more-verification-for-source-url)